### PR TITLE
[refactor](nereids)use nereids to implement TableQueryPlanAction

### PR DESCRIPTION
**Pull request by starocean999**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by starocean999** [comment]
run buildall

**Comment by starocean999** [comment]
run buildall

**Comment by morrySnow** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/TableQueryPlanAction.java [hunk 1]
```diff
@@ -163,65 +170,78 @@ public Object query_plan(
      */
     private void handleQuery(ConnectContext context, String requestDb, String requestTable, String sql,
             Map<String, Object> result) throws DorisHttpException {
-        // use SE to resolve sql
-        StmtExecutor stmtExecutor = new StmtExecutor(context, new OriginStatement(sql, 0), false);
+        List<StatementBase> stmts = null;
         try {
-            TQueryOptions tQueryOptions = context.getSessionVariable().toThrift();
-            // Conduct Planner create SingleNodePlan#createPlanFragments
-            tQueryOptions.num_nodes = 1;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/TableQueryPlanAction.java:171]: this is used to ensure generate one fragment plan. and will be check fragment num later. so in nereids, we should set enable_parallel_result_sink = true

**Comment by starocean999** [comment]
run buildall

**Comment by starocean999** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by 924060929 — APPROVED** [review]

**Pull request merged**